common/src/main/java/com/wynntils/features/overlays/TrackerOverlayFeature.java

```diff
@@ -40,35 +38,23 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @ConfigCategory(Category.OVERLAYS)
-public class QuestInfoOverlayFeature extends Feature {
+public class TrackerOverlayFeature extends Feature {
```

Comment:
If you decide to rename the model, follow on here.

I am also not confident the feature separation is needed, but I can understand it.

Do you want to add a config upfixer so people's config don't get lost?

Reply:
I'm a bit more gung-ho about dropping configuration as long as we call ourselves alpha. :-) If it is like a simple one-liner I might add an upfixer, otherwise I'm not sure I could be bothered to do this. :) If you want to contribute one, please go ahead.

Reply:
I added a generic upfixer for plain renamed configs, hoping it will be easier to add similar changes in the future.